src/index.js

```diff
@@ -41,66 +25,50 @@ export default {
   },
   methods: {
     makeBaseUrl() {
-      const { baseUrl, environment } = this;
-      if (baseUrl != null) {
+      const { baseUrl } = this;
+
+      if (baseUrl) {
         return baseUrl;
       }
-      switch (environment) {
-        case "sandbox":
-          return "https://verify.sandbox.berbix.com";
-        case "staging":
-          return "https://verify.staging.berbix.com";
-        default:
-          return "https://verify.berbix.com";
-      }
+
+      return "https://verify.berbix.com";
     },
     frameUrl() {
       const {
-        overrideUrl,
-        version,
-        clientId,
-        role,
-        templateKey,
-        email,
-        phone,
-        continuation,
         clientToken,
+        overrideUrl,
+        showCloseModalButton,
         showInModal,
+        version,
       } = this;
-      if (overrideUrl != null) {
+
+      if (overrideUrl) {
         return overrideUrl;
       }
-      const token = clientToken || continuation;
-      const template = templateKey || role;
+
+      const token = clientToken;
       var options = ["sdk=BerbixVue-" + SDK_VERSION];
-      if (clientId) {
-        options.push("client_id=" + clientId);
-      }
-      if (template) {
-        options.push("template=" + template);
-      }
-      if (email) {
-        options.push("email=" + encodeURIComponent(email));
-      }
-      if (phone) {
-        options.push("phone=" + encodeURIComponent(phone));
-      }
+
       if (token) {
         options.push("client_token=" + token);
       }
+
       if (showInModal) {
-        options.push("modal=true");
+        options.push(`modal=${showCloseModalButton ? 2 : 1}`);
```

Comment:
For understandability, could you make this an enum? (and by enum, I mean an object since this is JS?)